src/AnalysisServices/AnalysisServices.Dataplane/Models/AsAzureDataplaneClient.cs

```diff
@@ -0,0 +1,231 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
src/AnalysisServices/AnalysisServices.Dataplane/Models/AsAzureHttpClient.cs is replaced by this one and should be removed, right?

Reply:
The IAsAzureHttpClient interface is implemented by AsAzureDataplaneClient, but the implementation of the interface in the AsAzureHttpClient.cs file is unused. I added a commit which removes this unused implementation.